MultiSlider.js

```diff
@@ -410,15 +439,30 @@ export default class MultiSlider extends React.Component {
               ref={component => this._markerTwo = component}
               {...this._panResponderTwo.panHandlers}
             >
-              <Marker
-                pressed={this.state.twoPressed}
-                markerStyle={this.props.markerStyle}
-                pressedMarkerStyle={this.props.pressedMarkerStyle}
-                currentValue={this.state.valueTwo}
-                enabled={this.props.enabledTwo}
-                valuePrefix={this.props.valuePrefix}
-                valueSuffix={this.props.valueSuffix}
-              />
+                {isMarkersSeparated===false &&
```

Comment:
spaces around ===

Reply:
Updated in last commit:
https://github.com/sadeghipour/react-native-multi-slider/blob/master/MultiSlider.js#L442